app/services/analytics.rb

```diff
@@ -12,8 +12,19 @@ def track_event(event, attributes = {})
       event_properties: attributes.except(:user_id),
       user_id: attributes[:user_id] || user.uuid,
     }
-    ahoy.track(event, analytics_hash.merge!(request_attributes))
+    analytics_hash.merge!(request_attributes)
+
+    ahoy.track(event, analytics_hash)
     register_doc_auth_step_from_analytics_event(event, attributes)
+
+    # Tag NewRelic APM trace with a handful of useful metadata
+    # https://www.rubydoc.info/github/newrelic/rpm/NewRelic/Agent#add_custom_attributes-instance_method
+    ::NewRelic::Agent.add_custom_attributes(
```

Comment:
Sometimes we track multiple events per page load, will this just take the most recent event?

Reply:
Correct - Last wins.   Not ideal, but without breaking into spans, does it seem reasonable-ish as a quick+dirty solution?

Reply:
Yup! Just wanted to check expectations -- this isn't going to replace our current logging infra, just augmenting some existing stuff with details, etc etc